libfreerdp/core/info.c

```diff
@@ -534,11 +534,9 @@ static BOOL rdp_read_info_packet(rdpRdp* rdp, wStream* s)
 			WLog_ERR(TAG, "failed to convert Domain string");
 			return FALSE;
 		}
-
-		Stream_Seek(s, cbDomain);
 	}
 
-	Stream_Seek(s, 2);
+	Stream_Seek(s, cbDomain + 2);
```

Comment:
Changes here not really required (don´t change anything)

Reply:
Agree. Since in `rdp_write_info_packet` function, we firstly write the content and then the null terminator, it is better to use the original code which reads the stream separately for content and terminator.

I will revert this part.